packages/app/package.json

```diff
@@ -8,9 +8,6 @@
   "scripts": {
     "MAIN COMMANDS": "=========================================================",
     "develop": "cross-env START_HOT=1 npm run dev:start:renderer",
-    "pack": "npm run build && electron-builder --dir --publish never",
-    "release": "electron-builder --publish always --mac --win --linux",
-    "sign-dev-electron-app": "codesign --deep --force --verbose --sign - node_modules/electron/dist/electron.app",
```

Comment:
@igorbezsmertnyi is one of the above commands used in CI? 🤷‍♂️

Reply:
nope